.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
```

Comment:
As I'd mentioned in the next comment, if make already runs unit tests, then that alone should be sufficient. Thee only reason I don't like invoking `make unit-tests` explicitly is that if the `Makefile` is changed over time so that `all` doesn't depend on `unit-tests`, we'd lose this functionality. 

> does && actually do a boolean and on the exit code?

`&&` helps when commands are being chained in that if there's any error with the command preceding `&&`, anything proceeding it will not be executed. With ';' you'd silently consume it. Also see [this section in travis docs](https://docs.travis-ci.com/user/customizing-the-build#Customizing-the-Build-Step) for an example of the same.